src/Microsoft.DocAsCode.Common/YamlMime.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Common
+{
+    using System.IO;
+
+    public static class YamlMime
+    {
+        public const string ManagedReference = nameof(ManagedReference);
+        public const string TableOfContent = nameof(TableOfContent);
+        public const string XRefMap = nameof(XRefMap);
+
+        public static string ReadMime(TextReader reader)
+        {
+            var line = reader.ReadLine();
+            if (!line.StartsWith("#"))
+            {
+                return null;
+            }
+            return line.TrimStart('#', ' ');
```

Comment:
if so, need add some comments for the parameter "comments" in YamlUtility that its first line would be used as mime?

---
In reply to: [69884493](https://github.com/dotnet/docfx/pull/476#discussion_r69884493) [](ancestors = 69884493)